content/post/imbalanced_data.Rmd

```diff
@@ -0,0 +1,238 @@
+---
+title: mlr3 imbalanced data
+author: Giuseppe Casalicchio
+date: '2020-03-30'
+slug: mlr3-imbalanced
+categories: []
+tags: ['mlr3', 'mlr3pipelines', 'mlr3pipelines', 'basics', 'visualization']
+packages: ['mlr3', 'mlr3learners', 'mlr3pipelines', 'mlr3tuning']
+---
+
+# Intro 
+
+This use case compares different approaches to handle class imbalancy for the [`optdigits`](https://www.openml.org/d/980) binary classification data set using the mlr3 package. 
+We mainly focus on undersampling the majority class, oversampling the minority class, and the smote imbalancy correction that enriches the minority class with synthetic data. 
+The use case requires prior knowledge in basic ML concepts (e.g., issues imbalanced data, hyperparameter tuning, nested cross-validation) and `mlr3`, `mlr3pipelines` and `mlr3tuning` is required.
+You can find most of the content here also in the [mlr3book](https://mlr3book.mlr-org.com/) in a more detailed way.
+
+The steps are performed:
+
+* Retrieve data sets from `OpenML`
+* Define imbalancy correction pipelines (undersampling, oversampling and smote) with `mlr3pipelines`  
+* Combine pipeline with a learner and autotune it with `mlr3tuning`
+* Benchmark the autotuned pipelines and visualize the results using `mlr3viz`
+
+# Prerequisites
+
+## Loading required packages
+
+```{r}
+library("mlr3")          # mlr3 base package
+library("mlr3misc")      # contains some helper functions
+library("mlr3measures")  # lots of performance measures
```

Comment:
I would avoid this line. See readme: https://github.com/mlr-org/mlr3measures

It does not clash with calls in this post but it is recommended practice.